jestGlobalSetup.js

```diff
@@ -0,0 +1,5 @@
+const aggregateTranslations = require('@cerner/terra-aggregate-translations');
```

Comment:
You don't need to worry about pulling in or running terra-aggregate translations in jest if you use the jest-config-terra package: https://engineering.cerner.com/terra-ui/dev_tools/cerner/jest-config-terra/about